packages/realm-server/server.ts

```diff
@@ -423,6 +424,24 @@ export class RealmServer {
     return realms;
   }
 
+  private sendEvent = async (user: string, eventName: string) => {
+    let dmRooms =
+      (await this.matrixClient.getAccountData<Record<string, string>>(
+        'boxel.session-rooms',
+      )) ?? {};
+    let roomId = dmRooms[user];
+    if (!roomId) {
+      console.error(
+        `Failed to send event: ${eventName}, cannot find session room for user: ${user}`,
+      );
+    }
+
+    await this.matrixClient.sendEvent(roomId, 'm.room.message', {
+      body: `${eventName}`,
```

Comment:
```suggestion
      body: `{ eventType: ${eventName} }`,
```

This would let us add additional payload in the future without breaking existing handler code